### DISCUSSION TITLE
Abbreviations: first time long

### DISCUSSION THREAD

**Discussion by jan-david-fischbach** [executablebooks/meta#1056] [Q&A] [answered]
Hi is there any way to use the abbreviations in the front matter such that the first occurrence introduces the long form? When converting to latex (and PDF) are the abbreviations also considered?

**Comment by jan-david-fischbach** [comment, marked as answer]
This is now available via `firstTimeLong` in the abbreviations engine

**Reply by rowanc1** [reply]
Thanks for your PR here!!
